marklogic-data-hub-central/ui/src/pages/Browse.tsx

```diff
@@ -321,27 +318,11 @@ const Browse: React.FC<Props> = ({location}) => {
           await setZeroStateQueryOptions();
           let preferencesObject = {
             ...parsedPreferences,
-            zeroState: searchOptions.zeroState
           };
           updateUserPreferences(user.name, preferencesObject);
         }
       } else {
-        if (!parsedPreferences.zeroState && searchOptions.zeroState) {
-          let options: any = {
-            searchText: parsedPreferences.query.searchText || "",
-            entityTypeIds: parsedPreferences.query.entityTypeIds || [],
-            selectedFacets: parsedPreferences.query.selectedFacets || {},
-            selectedQuery: searchOptions.selectedQuery || "select a saved query",
-            sidebarQuery: searchOptions.sidebarQuery,
-            start: parsedPreferences.start || 1,
-            pageNumber: parsedPreferences.pageNumber || 1,
-            pageLength: parsedPreferences.pageLength,
-            propertiesToDisplay: searchOptions.selectedTableProperties || [],
-            zeroState: parsedPreferences.zeroState,
-            sortOrder: parsedPreferences.sortOrder || [],
-            database: parsedPreferences.database
-          };
-          await setPageQueryOptions(options);
+        if (!parsedPreferences.zeroState) {
```

Comment:
I still see few zeroState specific conditions like this here. This condition seems irrelevant, now that there is no zeroState page. It would be better if we remove all such references of zeroState based variables and conditions.